rasa/utils/tensorflow/models.py

```diff
@@ -103,6 +103,10 @@ def fit(
     ) -> None:
         """Fit model data"""
 
+        if not silent:
```

Comment:
Why do we need this `if` construct? Isn't `fit()` called only during training?
Also, if its necessary then we should rename silent to something more meaningful?